src/coder/graph.py

```
@@ -95,6 +96,7 @@ async def __call__(self, state: State) -> dict:
         messages_after_invoke = await llm.bind_tools(self.github_tools).ainvoke(
             messages
         )
+        print(messages_after_invoke)
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)